src/aotcompile.cpp

```diff
@@ -836,6 +844,8 @@ void *jl_get_llvmf_defn(jl_method_instance_t *mi, size_t world, char getwrapper,
             // if compilation succeeded, prepare to return the result
             if (optimize)
                 PM->run(*m.get());
+            else
+                PM_minimal->run(*m.get());
```

Comment:
I think it would make more sense to run this during printing when we strip out the other metadata (controlled by the 'raw' flag). Thoughts?

Reply:
Yeah, that seems like a better place to schedule that. I'll create a PR.